src/vue/backend/src/views/guided-setup/Api.vue

```diff
@@ -63,7 +62,7 @@ export default {
             this.$router.go(-1)
         },
         goSkipStep: function () {
-            this.$router.push({ path: '/guided-setup/connect/general' })
+            this.$router.push({ path: '/guided-setup/general' })
```

Comment:
We will want the connect piece to visually separate the major steps in the set up